ios/RNCardscan.m

```diff
@@ -35,8 +35,8 @@ - (void)userDidScanCard:(ScanViewController * _Nonnull)scanViewController credit
         @"action" : @"scanned",
         @"payload": @{
             @"number": number,
-            @"expiryMonth": expiryMonth,
-            @"expiryYear": expiryYear
+            @"expiryMonth": expiryMonth ?: [NSNull null],
+            @"expiryYear": expiryYear ?: [NSNull null]
```

Comment:
qq: Do we need to update these objc files if we change the swift protocols?

Reply:
Hmm, I'm not very familiar with Swift but you might have to update this bridge file if any if `number`, `expiryMonth`, or `expiryYear` changes although I don't see that happening a lot.

Reply:
hmmm i see. how do i update the file for example if we decide to break interface and change the variable name `number` to `last4`. would i have to do it manually?

Reply:
Yup, this method takes in the CreditCard object from Swift so you will have to update https://github.com/getbouncer/cardscan-ios-react/pull/3/files#diff-7a94af744d0384802abe0b7234d25ce3R30 above appropriately and this payload block is what we send back to React Native https://github.com/getbouncer/cardscan-ios-react/blob/master/example/App.js#L25